src/Get.test.tsx

```diff
@@ -101,7 +101,7 @@ describe("Get", () => {
     });
   });
 
-  describe("composing nested urls", () => {
+  fdescribe("composing nested urls", () => {
```

Comment:
Careful! Please don't skip all the tests 😉

Reply:
😱 Thank you for catching that 🙈